cli-feed-v4.json

```diff
@@ -7172,6 +7172,122 @@
           "default": "true"
         }
       ]
+    },
+    "3.39.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "netcore3": {
+            "displayInfo": {
+              "displayName": ".NET Core 3.1",
```

Comment:
I changed this from "3" to "3.1" per this comment: https://github.com/Azure/azure-functions-tooling-feed/pull/347#issuecomment-1124488372

I think the only reason it didn't happen when 3.39.0 was generated is because releases 3.37.0 and 3.38.0 are out of order

FYI @soninaren let me know if any of that sounds wrong

Reply:
This should be okay, no concern from my end!